source/db/en-projects-rejected.ls

```diff
@@ -290,14 +290,14 @@ projects-rejected =
     name: "Facebook"
     categories: [
       * name: "Web Services"
-        subcategories: ["Email Accounts", "Instant Messaging", "Social Networks", "Video & Voice"]
+        subcategories: ["Email Accounts", "Social Networks", "Video & Voice"]
       * name: "Servers"
-        subcategories: ["Email Server", "Instant Messaging", "Social Networks", "Video & Voice"]
```

Comment:
Ah sorry @vyp, thanks for merging this. Yes, I think it's better to err on the side of having less projects on the page, and I don't think anyone will really pay too much attention to the details we're discussing here :thumbsup: